docs/configure.md

```diff
@@ -197,7 +202,8 @@ Available elements are:
 | ResultsDirectory  | string | Directory for test run reports. E.g. trx, coverage etc.                                         |
 | SolutionDirectory | string | Working directory for test invocation. Results directory can be relative to this. Used by IDEs. |
 | MaxCpuCount       | int    | Degree of parallelization, spawns `n` test hosts to run tests. Default: 1. Max: Number of cpu cores. |
-| TestSessionTimeout | int   | Testplatform will cancel the test run after it exceeded given TestSessionTimeout and will show the results of tests which ran till that point. **This setting is available from Visual Studio 2017 Update5 onwards.** |
+| TestSessionTimeout | int   | Testplatform will cancel the test run after it exceeded given TestSessionTimeout in milliseconds and will show the results of tests which ran till that point. **This setting is available from Visual Studio 2017 Update 5 onwards.** |
```

Comment:
Updated to `Required version 15.5+`. Not mentioning it as vstest.console, VS or testhost.